docs/static/spec/rasa.yml

```
@@ -880,16 +880,19 @@ components:
     200Story:
       description: Success
       content:
-        text/markdown:
+        yml:
```

I think this refers to the mime type, so I would suggest to change it to
```suggestion
        text/yaml:
```

---

Will do, I wanted to get your opinion if I'm getting the yml format right first. I will update the others now. :D